content/en/docs/tasks/access-application-cluster/connecting-frontend-backend.md

```diff
@@ -11,34 +11,23 @@ microservice. The backend microservice is a hello greeter. The
 frontend and backend are connected using a Kubernetes
 {{< glossary_tooltip term_id="service" >}} object.
 
-
-
-
 ## {{% heading "objectives" %}}
 
-
 * Create and run a microservice using a {{< glossary_tooltip term_id="deployment" >}} object.
 * Route traffic to the backend using a frontend.
 * Use a Service object to connect the frontend application to the
   backend application.
 
-
-
-
 ## {{% heading "prerequisites" %}}
 
-
-* {{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
```

Comment:
nit: line 25, Do you need a list at line 25?

Reply:
rechecked the shortcode ...
you are right, this shortcode is meant to be used as a standalone paragraph and it already contains a list.